torch/_inductor/runtime/triton_heuristics.py

```diff
@@ -470,7 +475,9 @@ def _precompile_config(self, cfg: Config, warm_cache_only: bool):
             device_interface.synchronize(device_interface.current_device())
 
             try:
+                start_time = time.time_ns()
                 binary = triton.compile(*compile_args, **compile_kwargs)
+                compile_time_ns = time.time_ns() - start_time
```

Comment:
Because you don't report these stats, except incorporating them into cache products, it is impossible to observe the cache hit performance of triton.compile (which is surely nonzero, because it has to go actually compute cache keys and then do the lookup.)